### DISCUSSION TITLE
Turning annotations on and off

### DISCUSSION THREAD

**Discussion by dan-goodrick** [SkalskiP/make-sense#251] [Q&A] [answered]
I looked through the docs and through the issues but didn't see any discussion of this topic.  In my current task, I end up with a bunch of labels that make it hard to see the details of the underlying image. It would be nice if there were a way to turn the bounding boxes on and off (while a hot key is pressed) and if I could also turn on and off selected labels (maybe a check box next to a label list).

**Comment by SkalskiP** [comment, marked as answer]
Hello, @dan-goodrick 👋 ! It seems like a very good idea. Let, let me create an issue based on that discussion. I also think it should be fairly easy to implement. So I'll try to do it relatively fast.
